content/monitors/_index.md

```diff
@@ -68,14 +68,13 @@ Setting the above to **Notify** send an email for the monitor audit events to al
 
 ## Manually resolve your monitor
 
-It only makes sense in a couple cases to manually resolve your monitor:
+The monitor *Resolve* function is artificially switching the monitor status to `OK` for its next evaluation. The following monitor evaluation will be performed normally on the data the monitor is based on.
 
-* If the monitor is in a "no data" state then resolving it hides it from the triggered monitors page.
-* If the monitor is in the triggered state but has stopped reporting data then resolving it hides it from the triggered monitors page.
+If a monitor is alerting because its current data corresponds to its `ALERT` state, *Resolve* will have the monitor to follow the following state switch `ALERT -> OK -> ALERT`. Thus it's not appropriate for acknowledging that you have seen the alert or telling Datadog to ignore the alert.
 
-Otherwise the monitor picks up the current state on the next evaluation.
+Manually *Resolve* a monitor is a good fit to cases where data is reported intermittently: after triggering an alert, the monitor doesn't receive further data so it cannot evaluate the alerting conditions anymore and recover to the `OK` state. In that case the *Resolve* function or the *Automatically resolve monitor after X hours* switch the monitor back to OK.
```

Comment:
Manually *Resolve*ing a monitor is appropriate for cases where data is reported intermittently: after triggering an alert, the monitor doesn't receive further data so it can no longer evaluate alerting conditions and recover to the `OK` state. In that case the *Resolve* function or the *Automatically resolve monitor after X hours* switches the monitor back to OK.